extension/src/lib/chromeSettings.js

```diff
@@ -0,0 +1,80 @@
+/*globals fluid, gpii */
+
+"use strict";
+
+fluid.defaults("gpii.chrome.settings", {
+    gradeNames: "fluid.modelComponent",
+    components: {
+        chromeVox: {
+            type: "gpii.chrome.extensionHolder",
+            options: {
+                extensionId: "kgejglhpjiefppelpmljglcjbhoiplfn",
+                model: {
+                    extensionEnabled: "{settings}.model.currentSettings.screenReaderTTSEnabled"
+                }
+            }
+        },
+        highContrast: {
+            type: "gpii.chrome.highContrast",
+            options: {
+                model: {
+                    highContrastEnabled: "{settings}.model.currentSettings.highContrastEnabled",
+                    highContrastTheme: "{settings}.model.currentSettings.highContrastTheme"
+                }
+            }
+        },
+        zoom: {
+           type: "gpii.chrome.zoom",
+           options: {
+               model: {
+                   magnifierEnabled: "{settings}.model.currentSettings.magnifierEnabled",
+                   magnification: "{settings}.model.currentSettings.magnification"
+              }
+           }
+        },
+        wsConnector: {
+            type: "gpii.wsConnector",
+            options: {
+                solutionId: "com.ilunion.cloud4chrome",  // This will change in the near future
+                flowManager: "ws://localhost:8081/browserChannel",
+                retryTime: 10
+            }
+        }
+    },
+    model: {
+        currentSettings: { // default values
+            screenReaderTTSEnabled: undefined,
+            onScreenKeyboardEnabled: undefined,
+            highContrastEnabled: false,
+            highContrastTheme: "",
+            invertColors: undefined,
+            greyscale: undefined,
+            magnifierEnabled: false,
+            magnification: 1
+        },
+        oldSettings: null
+    },
+    invokers: {
+        onSettingsChange: {
```

Comment:
This should probably be renamed. With the name starting with "on" it sounds more like an event. However, this could be used outside of the event system. Perhaps it should just be called "updateCurrentSettings" or something like that.